src/SessionMessages/SessionMessage/SessionMessage.tsx

```diff
@@ -41,17 +42,31 @@ export const SessionMessage: FC<SessionMessageProps> = ({
   children
 }) => {
   const { theme, isLoading } = useContext(ChatContext);
+  const hasFollowUpResponse = !!conversation.followUpResponse;
 
   return (
     <motion.div key={conversation.id} variants={messageVariants}>
       <Card className={cn(theme.messages.message.base)}>
         {children || (
           <>
-            <MessageQuestion question={conversation.question} files={conversation.files} />
-            <MessageResponse
-              response={conversation.response}
-              isLoading={isLast && isLoading}
+            <MessageQuestion
+              question={conversation.question}
+              files={conversation.files}
             />
+
+            {hasFollowUpResponse ? (
+              <MessageResponseRecommended
+                response={conversation.response}
+                followUpResponse={conversation.followUpResponse}
+                isLoading={isLast && isLoading}
+              />
+            ) : (
+              <MessageResponse
+                response={conversation.response}
+                isLoading={isLast && isLoading}
+              />
+            )}
```

Comment:
I think here instead of an either, or we would always show the `<MessageResponse />` and optionally show `<FollowUpResponse />` if some exist

Reply:
@steppy452 Thank you for clarifying the requirements. Please review the changes I have made. Also, regarding the image, what are we expecting the image payload to be? Currently, I have it as a URL.